README.md

```diff
@@ -10,6 +10,7 @@ The following table shows the supported devices and applications:
 | STM32F4-Discovery                      | &#128504; |  &#128504;  |
 | Raspberry Pi 2                         | &#128504; |  &#128504;  |
 | Artik053                               | &#128504; |  &#128504;  |
+| Artik530                               |  |  |
```

Comment:
Please modify the Artik053, Artik530 labels to ARTIK 053 and ARTIK 530.
If IoT.js is supported on ARTIK 530, it should be marked in the table.

Reply:
If #109 is done, I will check it. I will change the label.